editor.planx.uk/src/components/Header.tsx

```diff
@@ -259,17 +272,21 @@ const PublicToolbar: React.FC<{
         ) : (
           <Breadcrumbs route={route} handleClick={navigate}></Breadcrumbs>
         )}
-        {showCenteredServiceTitle && <ServiceTitle />}
-        <IconButton
-          color="secondary"
-          onClick={handleRestart}
-          aria-label="Restart Application"
-          size="large"
-        >
-          <Reset color="secondary" />
-        </IconButton>
+        {showCentredServiceTitle && <ServiceTitle />}
+        {showResetButton ? (
+          <IconButton
+            color="secondary"
+            onClick={handleRestart}
+            aria-label="Restart Application"
+            size="large"
+          >
+            <Reset color="secondary" />
+          </IconButton>
+        ) : (
+          <Box />
```

Comment:
This is a bad hack to handle keeping the title centred - planning to revisit this!